minerl/herobraine/hero/handlers/agent/observations/inventory.py

```diff
@@ -2,14 +2,45 @@
 # Author: William H. Guss, Brandon Houghton
 
 import logging
+from typing import List, Sequence
 
-import jinja2
+from minerl.herobraine.hero.handlers import util
 from minerl.herobraine.hero.handlers.translation import TranslationHandler
 import numpy as np
 from minerl.herobraine.hero import spaces
 import minerl.herobraine.hero.mc as mc
 
 
+def _univ_obs_get_all_inventory_slots(obs: dict) -> List[dict]:
+    """
+    Observations in univ.json contain "slot dicts" containing info about
+    items held in player or container inventories. This function processes an obs dict
+    from univ.json, and returns
+    the list of "slot" dictionaries, where every non-empty dictionary corresponds to
+    an item stack.
+
+    See the following link for an example format:
+    https://gist.github.com/shwang/9c8815e952eb2a4c308aea09f112cd6a#file-univ-head-json-L162
```

Comment:
In this gist it looks like item-stacks of air are also represented as dictionaries?